plugins/extractors/maxcompute/client.go

```diff
@@ -0,0 +1,66 @@
+package maxcompute
+
+import (
+	"context"
+
+	"github.com/aliyun/aliyun-odps-go-sdk/odps"
+	"github.com/aliyun/aliyun-odps-go-sdk/odps/account"
+	"github.com/aliyun/aliyun-odps-go-sdk/odps/tableschema"
+)
+
+type MCClient struct {
+	client  *odps.Odps
+	project *odps.Project
+}
+
+func NewMaxComputeClient(config Config) *MCClient {
+	aliAccount := account.NewAliyunAccount(config.AccessKey.ID, config.AccessKey.Secret)
+	client := odps.NewOdps(aliAccount, config.EndpointProject)
+	client.SetDefaultProjectName(config.ProjectName)
+
+	project := client.Project(config.ProjectName)
+
+	return &MCClient{
+		client:  client,
+		project: project,
+	}
+}
+
+func (c *MCClient) ListSchema(context.Context) (schemas []*odps.Schema, err error) {
+	err = c.project.Schemas().List(func(schema *odps.Schema, err2 error) {
+		if err2 != nil {
+			err = err2
+			return
```

Comment:
according to this behavior, please log the error so we can still know why some tables failed

Reply:
addressed